### PR TITLE
Adding rawData script input for k8s addons

### LINKED ISSUE #3811: Feature Request: Enable configuration of kube-proxy
**Is this an ISSUE or FEATURE REQUEST?** (choose one): FEATURE REQUEST

---
As a Kubernetes user, I would like to be able to further configure the kube-proxy `DaemonSet`. As an example, I need to set a toleration on kube-proxy so it runs on tainted nodes.

### PR DESCRIPTION
**Pull request by tariq1890**
fixes #3811

### CODE REVIEW

File: pkg/acsengine/testdata/addons/kubernetes-kube-proxy.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
...y9kbnNtYXNxLW5hbm55CiAgICAgIC0gbmFtZTogc2lkZWNhcgogICAgICAgIGltYWdlOiA8a3ViZXJuZXRlc0ROU1NpZGVjYXJTcGVjPgogICAgICAgIGxpdmVuZXNzUHJvYmU6CiAgICAgICAgICBodHRwR2V0OgogICAgICAgICAgICBwYXRoOiAvbWV0cmljcwogICAgICAgICAgICBwb3J0OiAxMDA1NAogICAgICAgICAgICBzY2hlbWU6IEhUVFAKICAgICAgICAgIGluaXRpYWxEZWxheVNlY29uZHM6IDYwCiAgICAgICAgICB0aW1lb3V0U2Vjb25kczogNQogICAgICAgICAgc3VjY2Vzc1RocmVzaG9sZDogMQogICAgICAgICAgZmFpbHVyZVRocmVzaG9sZDogNQogICAgICAgIGFyZ3M6CiAgICAgICAgLSAtLXY9MgogICAgICAgIC0gLS1sb2d0b3N0ZGVycgogICAgICAgIC0gLS1wcm9iZT1rdWJlZG5zLDEyNy4wLjAuMToxMDA1MyxrdWJlcm5ldGVzLmRlZmF1bHQuc3ZjLjxrdWJlcm5ldGVzS3ViZWxldENsdXN0ZXJEb21haW4+LDUsU1JWCiAgICAgICAgLSAtLXByb2JlPWRuc21hc3EsMTI3LjAuMC4xOjUzLGt1YmVybmV0ZXMuZGVmYXVsdC5zdmMuPGt1YmVybmV0ZXNLdWJlbGV0Q2x1c3RlckRvbWFpbj4sNSxTUlYKICAgICAgICBwb3J0czoKICAgICAgICAtIGNvbnRhaW5lclBvcnQ6IDEwMDU0CiAgICAgICAgICBuYW1lOiBtZXRyaWNzCiAgICAgICAgICBwcm90b2NvbDogVENQCiAgICAgICAgcmVzb3VyY2VzOgogICAgICAgICAgcmVxdWVzdHM6CiAgICAgICAgICAgIG1lbW9yeTogMjBNaQogICAgICAgICAgICBjcHU6IDEwbQogICAgICBkbnNQb2xpY3k6IERlZmF1bHQgICMgRG9uJ3QgdXNlIGNsdXN0ZXIgRE5TLgogICAgICBzZXJ2aWNlQWNjb3VudE5hbWU6IGt1YmUtZG5zCiAgICAgIG5vZGVTZWxlY3RvcjoKICAgICAgICBiZXRhLmt1YmVybmV0ZXMuaW8vb3M6IGxpbnV4"
```
Comment (jackfrancis) [inline pkg/acsengine/testdata/addons/kubernetes-kube-proxy.json]: For this example let's put in a fully static kube-proxy manifest (no `<varToReplace>` patterns).

**Comment by jackfrancis** [comment]
added a small comment, otherwise /lgtm

**Comment by pidah** [comment]
@tariq1890 this is great ! I also want to override the pod security YAML definition which is part of _kubernetesManifestSettingsInit_  https://github.com/Azure/acs-engine/blob/master/pkg/acsengine/artifacts.go#L239-L272 Any thoughts on this ? thanks @jackfrancis

**Comment by tariq1890** [comment]
@pidah Could you raise a GH issue for this? Happy to look into this :).

**Comment by pidah** [comment]
Thanks ! @tariq1890  I have raise an issue https://github.com/Azure/acs-engine/issues/3906

**Pull request merged**